docs/concepts/hosts.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "Hosts"
+icon: "ti-anchor" # themify icon pack : https://themify.me/themify-icons
+description: "The runtime environment for actors and capability providers"
+sidebar_position: 2
+type: "docs"
+---
+
+## Overview
+
+The wasmCloud **host** is an application runtime. The host is responsible for loading, instantiating, and communicating among WebAssembly components and capability providers, in a secure, distributed way.
+
+### Cattle, Not Pets
```

Comment:
I'd say this should be de-emphasized -- "cattle not pets" is almost default in today's culture when it matters, and is kind of also not up to us (someone could use wasmCloud very much as a pet)

Reply:
I think aligning to those culture norms is a good thing (i.e. wasmCloud works just how you are used to). There might be a better way to phrase it that won't make Peta mad, though.

Reply:
I like us being opinionated. And as the resident non-meat-eater, I'm not mad 🙂 Gonna decline this change